src/Client/Components/Layout/SiteFooter.razor

```diff
@@ -85,24 +85,22 @@
             <div class="col" id="footer-learn-col">
                 <div class="pb-2 font-weight-bold">Learn</div>
                 <div><Href href="https://start.steeltoe.io/" NewWindow="true">Steeltoe Initializr</Href></div>
-                <div><Href href="@(docsSite.GuidesHome)">Guides</Href></div>
-                <div><Href href="@(docsSite.DocsHome)">Documentation</Href></div>
-                <div><Href href="@(docsSite.ApiBrowserHome)">API Browser</Href></div>
-                <div><Href href="@(docsSite.BlogHome)">Blog</Href></div>
+                <div><Href href="@(_docsSite.GuidesHome)">Guides</Href></div>
+                <div><Href href="@(_docsSite.DocsHome)">Documentation</Href></div>
+                <div><Href href="@(_docsSite.ApiBrowserHome)">API Browser</Href></div>
+                <div><Href href="@(_docsSite.BlogHome)">Blog</Href></div>
             </div>
             <div class="col" id="footer-projects-col">
                 <div class="pb-2 font-weight-bold">Projects</div>
                 <div><Href href="/app-configuration">Application Configuration</Href></div>
                 <div><Href href="/circuit-breakers">Circuit Breakers</Href></div>
-                @*<div><Href href="/cloud-management">Distributed Tracing</Href></div>*@
-                <div><Href href="/logging">Dynamic Logging</Href></div>
```

Comment:
Looking at https://www-steeltoe-preview-141.azurewebsites.net/app-configuration, the logging link is shown, but here it looks like it was deleted. Am I not looking at the latest version of the site or the commits?
Also, I'm wondering why all the rendered menu names are prefixed with "Steeltoe", is the Href component doing that?

Reply:
This comment was in the footer, the menu items all have "Steeltoe", it's not from the href component. I also made a change in the naming convention of the slots ('preview' -> 'pr') so that is probably why it was out of sync